.github/workflows/pre-merge.yml

```diff
@@ -9,12 +9,12 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: '16'
+          node-version: '18'
```

Comment:
can we switch to 20 ? (or do we have a ticket drafting what's to be done...)

Reply:
There is a dependency that is not (yet) compatible with 20... Nothing to be done yet, I guess?
See https://github.com/scality/vaultclient/actions/runs/10678954415/job/29597111571#step:4:14

Or I can add a resolution, it should work so we can use node20 in the CI

Reply:
Ok no it's a subdep of Arsenal, so I will just add a comment in the arsenal upgrade deps tickets. If the dep can be upgraded, then we need to update vaultclient.